application/assets/javascripts/components/language-switch-example.js

```diff
@@ -36,9 +36,9 @@ LanguageSwitchExample.prototype.handleClick = function (event) {
   var $target = event.target
   nodeListForEach(this.$module.querySelectorAll('.' + this.currentClassName), function ($option) {
     $option.classList.remove(self.currentClassName)
-    $option.querySelector('a').focus()
+    $option.querySelector('button').focus()
   })
-  this.$iframe.setAttribute('src', $target.getAttribute('href'))
+  this.$iframe.setAttribute('src', $target.getAttribute('data-link'))
```

Comment:
Although the Attribute functions are ok for data attributes, there's a better JS API. Take a look at https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes#javascript_access